SConstruct

```diff
@@ -54,10 +54,8 @@ TARGET_FLAGS = {
 def construct():
     ccflags = (
         ' -g -O2 -Wall -Wextra -Werror '
-        '-Wno-null-pointer-arithmetic '
         '-Wno-sign-compare '
-        '-Wno-unknown-warning-option '
```

Comment:
-Wno-unknown-warning-option is needed with clang:

error: unknown warning option '-Wno-maybe-uninitialized'; did you mean '-Wno-uninitialized'? [-Werror,-Wunknown-warning-option]